examples/cactus-check-connection-ethereum-validator/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "check-connection-ethereum-validator",
+  "private": true,
+  "scripts": {
+    "start": "node ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "debug": "nodemon --inspect ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "build": "npm run build-ts && npm run copy-static-assets && npm run copy-blp-config && npm run replace-blp-config-path",
+    "build-ts": "tsc -p ./tsconfig.json",
+    "tslint": "tslint -c tslint.json -p tsconfig.json './*.ts'",
+    "copy-static-assets": "ts-node copyStaticAssets.ts",
+    "copy-blp-config": "ts-node copyBLPConfig.ts",
+    "replace-blp-config-path": "ts-node replaceBLPConfigPath.ts",
+    "init-check-connection-ethereum-validator": "ln -s ../examples/cactus-check-connection-ethereum-validator/node_modules ../../dist/node_modules"
+  },
+  "dependencies": {
+    "@types/node": "^14.0.24",
+    "body-parser": "^1.19.0",
+    "cookie-parser": "~1.4.4",
+    "debug": "~2.6.9",
+    "ethereumjs-common": "^1.5.1",
+    "ethereumjs-tx": "^2.1.2",
+    "express": "~4.16.1",
+    "fabric-ca-client": "~1.4.0",
```

Comment:
@stepniowskipCould you use the 2.x Fabric SDK? The older ones have critical security vulnerabilities IIRC

Reply:
Fabric SDK was not needed, I forgot to update this file earlier.
I have updated package.json file with needed version entry and removed unnecessary dependencies.

Reply:
@StepniowskiP Great, thank you. Is there a reason why ts-node and typescript are needed with different versions from the one that's already pulled in as a dependency in the project root?

Reply:
There is no need to use different versions and I can safely delete typescript dependency. However when I delete ts-node dep I get permission denied error. I can narrow ts-node version so it will match the one in project root.

Reply:
@StepniowskiP What command do you run when you get this permission denied error and what is being denied access to? Could you paste in the logs from the terminal as well?

Reply:
I have this error while executing npm run build command on step: ts-node copyStaticAssets.ts

Here are logs from terminal:
sh: 1: ts-node: Permission denied
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! check-connection-ethereum-validator@1.0.0-rc.3 copy-static-assets: `ts-node copyStaticAssets.ts`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the check-connection-ethereum-validator@1.0.0-rc.3 copy-static-assets script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/stepniowskip/.npm/_logs/2021-12-22T10_45_54_804Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! check-connection-ethereum-validator@1.0.0-rc.3 build: `npm run build-ts && npm run copy-static-assets && npm run copy-blp-config && npm run replace-blp-config-path`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the check-connection-ethereum-validator@1.0.0-rc.3 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/stepniowskip/.npm/_logs/2021-12-22T10_45_54_864Z-debug.log

I think I might have problem with my environment (I am running Ubuntu 20.04 on WSL2)
In that case I think I can delete this dependency as well

Reply:
@StepniowskiP I agree, it sounds like maybe you ran an npm global install as root (something like `sudo npm install -g ts-node`) and then because of that the root user now owns the `ts-node` binary that's on the path of your `stepniowskip` user. (Just a guess, could be an infinite number of other things too).

Either way, maybe you could try uninstalling that global `ts-node` altogether and go with something like `yarn ts-node` or `npx ts-node` instead, I think that should resolve the permission problem and then the additional ts-node dependency could be removed which would be great.

Reply:
@petermetz I have finally managed to solve this issue. After many tries, uninstalling ts-node and reinstalling it with command: `npm install -g ts-node` fixed the issue

Reply:
@StepniowskiP Great! In that case is it possible now to remove the additional dependencies from the package itself?

Reply:
@petermetz Sure, they are already removed.

Reply:
@StepniowskiP Thank you, that's great!